pygmt/tests/test_clib_loading.py

```diff
@@ -64,18 +89,85 @@ def test_load_libgmt_with_a_bad_library_path(monkeypatch):
     assert check_libgmt(load_libgmt()) is None
 
 
-def test_clib_names():
+def test_load_libgmt_with_broken_libraries(monkeypatch):
```

Comment:
```suggestion
class TestLibgmtBrokenLibs:
```

Turn this big test into a class with multiple sub-tests (see https://docs.pytest.org/en/4.6.x/getting-started.html#group-multiple-tests-in-a-class). A lot more lines of code, but the advantage is that we can easily see which test broke exactly in the pytest log (if things break in the future).

```python-traceback
pygmt/tests/test_clib_loading.py::TestLibgmtBrokenLibs::test_two_broken_libraries PASSED [ 37%]
pygmt/tests/test_clib_loading.py::TestLibgmtBrokenLibs::test_load_brokenlib_invalidpath PASSED [ 43%]
pygmt/tests/test_clib_loading.py::TestLibgmtBrokenLibs::test_brokenlib_invalidpath_workinglib PASSED [ 50%]
pygmt/tests/test_clib_loading.py::TestLibgmtBrokenLibs::test_invalidpath_brokenlib_workinglib PASSED [ 56%]
pygmt/tests/test_clib_loading.py::TestLibgmtBrokenLibs::test_workinglib_brokenlib_invalidpath PASSED [ 62%]
pygmt/tests/test_clib_loading.py::TestLibgmtBrokenLibs::test_brokenlib_brokenlib_workinglib PASSED [ 68%]
```